main.go

```diff
@@ -183,41 +182,59 @@ func createRouterClock(db *bolt.DB, clock Clock) *mux.Router {
 }
 
 func InitDefaultAccounts(db *bolt.DB) {
+	newSchoolRequest := NewSchoolRequest{
+		School:    "test school",
+		FirstName: "test",
+		LastName:  "admin",
+		Email:     "test@admin.com",
+	}
+	_ = createNewSchool(db, newSchoolRequest, "123qwe")
+}
+
+func createNewSchool(db *bolt.DB, newSchoolRequest NewSchoolRequest, adminPassword string) error {
 
-	schoolId, err := FindOrCreateSchool(db, "test school", "no city", 0)
+	schoolId, err := FindOrCreateSchool(db, newSchoolRequest.School, "no city", 0)
 	if err != nil {
 		lgr.Printf("ERROR school does not exist: %v", err)
 	}
 
 	lgr.Printf("INFO test school id - %s", schoolId)
 
 	admin := UserInfo{
-		Name:        "test@admin.com",
-		Email:       "test@admin.com",
-		PasswordSha: EncodePassword("123qwe"),
-		FirstName:   "test",
-		LastName:    "admin",
+		Name:        newSchoolRequest.Email,
+		Email:       newSchoolRequest.Email,
+		PasswordSha: EncodePassword(adminPassword),
+		FirstName:   newSchoolRequest.FirstName,
+		LastName:    newSchoolRequest.LastName,
 		Role:        UserRoleAdmin,
 		SchoolId:    schoolId,
 	}
 
 	_, err = CreateSchoolAdmin(db, admin)
 	if err != nil {
 		lgr.Printf("ERROR school admin is not created: %v", err)
-		return
+		return err
 	}
 
-	admin.Role = UserRoleTeacher
-	admin.Name = "teacher@admin.com"
-	admin.Email = "teacher@admin.com"
+	tEmail := newSchoolRequest.Email[:1] + "." + newSchoolRequest.Email[1:]
```

Comment:
teacher for admin!

if you have to create a new user - need to have a unique email for them

I add a dot after 1st letter in the email - it makes email different from original - but all email-servers handle it and route emails to the same postbox

Reply:
not necessary since I will only make 1 teacher admin